### ISSUE TITLE
Support for a bug report or sysdump in calicoctl 

### ISSUE THREAD

**Issue by abasitt** [projectcalico/calico#8020]
We currently have a troubleshooting guide [here](https://docs.tigera.io/calico/latest/operations/troubleshoot/) that requires us to run several commands separately. It would be great if we could have a feature that allows us to run a single command that can generate a bug report or sysdump for offline troubleshooting. For example, something similar to what cilium offers [here](https://docs.tigera.io/calico/latest/operations/troubleshoot/)

**Comment by lwr20** [comment]
So something like this?  https://docs.tigera.io/calico/latest/reference/calicoctl/node/diags

**Comment by lwr20** [comment]
I've added the enhancement label, because it sounds like you'd want it to do more than it currently does.  Also "good first issue", because it feels like adding more diags to this existing function in calicoctl is a nice starter task.

**Comment by abasitt** [comment]
Thank you. Exactly something like on top of node diag. 
I actually played with node diag its good but its limited to a node. It will be great if we have a package which is cluster level e.g. cluster diag and that has below features.
- accept array of nodes to collect node diags from all the nodes
- run all the kubernetes commands from here as well https://docs.tigera.io/calico/latest/operations/troubleshoot/
- some level of possible filtering to limit the logs
tldr, the cluster diag package will contain multiple node diag plus useful kubernetes commands to check the overall health of cluster.
